deploy.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: extensions/v1beta1
```

Comment:
Let's put this file to separate directory  `deploy` or something.

Reply:
Fixed